lib/record-serialization.js

```diff
@@ -12,9 +12,11 @@ module.exports = jsrs;
 // one of our priorities to optimize it.
 const USE_NATIVE_SERIALIZER = false;
 
-const jstpNative =
-  safeRequire('../build/Release/jstp') ||
-  safeRequire('../build/Debug/jstp');
+let [error, jstpNative] = safeRequire('../build/Release/jstp');
+
+if (error) {
+  [error, jstpNative] = safeRequire('../build/Debug/jstp');
```

Comment:
Yeah, I'd rather only show the error that occurred trying to load the release build when both failed if we should only show only one. Alternatively, we may show both, like we do now. I'm fine with it both ways, slightly leaning towards the former one.